src/sentry/auth/helper.py

```diff
@@ -379,6 +383,25 @@ def _post_login_redirect(self) -> HttpResponseRedirect:
 
         return response
 
+    def handle_authentication_using_verification_key(self, identity, verification_key):
+        cluster = redis.clusters.get("default").get_local_client_for_key(_REDIS_KEY)
+        verification_value_byte = cluster.hgetall(verification_key)
```

Comment:
`verification_value_byte`? what does `byte` mean here?

Reply:
when i get the info from redis, the retrieved is a byte type object and couldn't be read from as is